rfcs/text/0008-threat-intel.md

```diff
@@ -23,30 +23,38 @@ Stage 1: Describe at a high level how this change affects fields. Which fieldset
 
 ### Proposed New Fields for Threat fieldset
 
-  * threat.time_first_seen
+  * threat.ioc.time_first_seen
     _The date and time when intelligence souce first reported sighting this indicator._
-  * threat.time_last_seen
+  * threat.ioc.time_last_seen
     _The date and time when intelligence source last reported sighting this indicator._
-  * threat.sightings
+  * threat.ioc.sightings
     _Number of times this indicator was observed conducting threat activity._
   * threat.type
     _Type of indicator as reprsented by Cyber Observable in STIX 2.0_
-  * threat.description
+  * threat.ioc.description
     _Describes the type of action conducted by the threat._
-  * threat.tlp
-    _Traffic Light Protocol, which dictates sharing policies_
-  * threat.classification
+threat.marking.tlp
```

Comment:
This should only appear in threat intel documents, and should be preserved in the matching and enrichment documents. 
The reason for nesting under `.marking` is to align it with STIX 2.0. Although `tlp` marking is used more widely in industry, government organizations often create their own marking nomenclature. If we use `.marking` it will allow support for those use cases.